src/sfctl/custom_deployment.py

```diff
@@ -0,0 +1,131 @@
+# -----------------------------------------------------------------------------

```

Comment:
It looks like this file doesn't have very much test coverage. Can you add some unit tests?

Reply:
took your suggestion to the merge utility code as this is no longer here

Reply:
Added more tests to improve coverage